### ISSUE TITLE
[Feature Request] Favourites Menu 

### ISSUE THREAD

**Issue by zero77** [SchizoDuckie/DuckieTV#617]
Can you get DuckieTV to show in the favourites menu weather tv shows have been cancelled, or finished and other things like that, sidereel.com usually shows you this sort of thing when you search for a show.

**Comment by Js41637** [comment]
Are you talking about when adding a show, when your in your own library you can select Filter and show different Status, Returning, Canceled or Ended.

**Comment by zero77** [comment]
Yes, that sort of thing but instead of filters it shows up on the tv show cover next to the amount of episodes you haven’t seen.

**Comment by Js41637** [comment]
I guess but it isn't ideal to clutter the posters with all those little details, how could you even convey whether the show was returning, ended, cancelled or airing?

**Comment by zero77** [comment]
Well I guess to ovoid cluttering then just add the important ones like returning, ended and cancelled and maybe, change the back ground colour or move them in to a group or something to make them stand out and appear different to show that they have ended etcetera.

**Comment by garfield69** [comment]
sidereel.com only shows these additional details when you are searching for a show.
That is similar to DuckieTV's *add-a-show* page. DuckieTV gives you those extra details on a sidepanel when you mouse over a poster. We like to keep the poster itself as uncluttered as possible.
![image](https://cloud.githubusercontent.com/assets/6933240/13727520/6839c11c-e95a-11e5-809e-5e7b85d30f34.png)
when you are looking at your own *tracked* shows in sidereel.com, there are just the posters, no details are present.
This is similar to to the DuckieTV's *favourites* page. On this page the *filter* option provides a way to reduce the list to show just returning,ended,production,cancelled or planned shows, as well as selecting one or more genre.
In addition, the next release can also sort the shows by name, added, first aired, and not-watched order.

In summary, I agree with Js41637 in that adding the show status (whether by overlaying the poster or by some other graphical means in the poster area) is going to make that page overloaded.
